corehq/pillows/mappings/xform_mapping.py

```diff
@@ -43,6 +43,10 @@
             "type": "date",
             "format": DATE_FORMATS_STRING
         },
+        "server_modified_on": {
```

Comment:
or maybe we do: https://github.com/dimagi/commcare-hq/blob/03251f4ae769f1c584586c800b7e928c059cbd3d/corehq/apps/hqcase/management/commands/reindex_sql_forms_in_domain.py#L22-L22